MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -34,8 +34,8 @@ DECLARE_SUBWINDOW(EnggDiffractionViewQtGUI)
 const double EnggDiffractionViewQtGUI::g_defaultRebinWidth = -0.0005;
 int EnggDiffractionViewQtGUI::m_currentType = 0;
 int EnggDiffractionViewQtGUI::m_currentRunMode = 0;
+bool EnggDiffractionViewQtGUI::m_fittingMutliRunMode = false;
```

Comment:
Note the typo: `Mutli` -> `Multi`.

Is this variable needed, or implicit in the number of runs that you'll have in the vector?

At first look it sounds like the kind of decision multi/single that should be done in the presenter / the presenter should be aware of.